### ISSUE TITLE
UIApplicationDelegate error in ios (not receiving notification)

### ISSUE THREAD

**Issue by shaharyar123** [fechanique/cordova-plugin-fcm#244]
FCM perfectly working in android but getting error when run on IOS. 

> <FIRMessaging/WARNING> Error none of the remote notification callbacks implemented by UIApplicationDelegate

what is this error about ? am i missing something ?
any answer or solution will be appreciated!

**Comment by Haideralee** [comment]
i having same issue but in my scenario, **Push Notification** receive successfully when APP is not open or APP is in background and if i add **FirebaseAppDelegateProxyEnabled** in **Info.plist**, set **NO** then i do not receive any notification on background or foreground but firebase warning appear on xcode logs.

> <FIRMessaging/WARNING> Error none of the remote notification callbacks implemented by 
UIApplicationDelegate

**Comment by alexandrunicolae** [comment]
any luck in finding a solution? thanks

**Comment by Haideralee** [comment]
@alexandrunicolae which IOS version are you testing ?

**Comment by alexandrunicolae** [comment]
@Haideralee 10.0.2

**Comment by Haideralee** [comment]
@alexandrunicolae did you upload ios development certificate on your project.

**Comment by alexandrunicolae** [comment]
@Haideralee yes, i get notifications when the app is in background or closed, the problem is when the app is in foreground or when I tap on a notification

**Comment by Haideralee** [comment]
@alexandrunicolae this behavior come to me on IOS < 10 version. please check this Opned issue https://github.com/fechanique/cordova-plugin-fcm/issues/247

**Comment by alexandrunicolae** [comment]
@Haideralee Solved it, I forgot to run cordova build ios and my app.js didn't update. so it works now. I didn't try <10 but found this https://github.com/martin-podlubny/cordova-plugin-fcm

**Comment by Haideralee** [comment]
@alexandrunicolae great

**Comment by saninaimayer** [comment]
I have the same error. My app receive and process notification in foreground but in background show this error: <FIRMessaging/WARNING> FIRMessaging receiving notification in invalid state 2.

My content-available set 1.

My iOS version is 10.3.3. 

Please help me.

{
aps = {
"content-available" = 1;
};
code = 053284;
"gcm.message_id" = "0:1504723579760179%bd2dffcfbd2dffcf";
message = "C\U00f3digo VEM 053284. Valor solicitado: 1.00";
type = "validation_code";
value = "1.00";
}
